UPGRADE.md

```diff
@@ -49,15 +49,15 @@ Exit codes
 
 Exit codes for `fix` command have been changed and are build using the following bit flags:
 
-1.x bit | 2.0 bit | Description                                                 | Note
--------:| -------:| ----------------------------------------------------------- | ----
```

Comment:
from what I see @julienfalque asked you to remove the `:` here

they are supposed to align columns with integer to right side. I would keep that alignment.